subsys/power/policy.c

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2018 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <kernel.h>
+#include <soc.h>
+#include "pm_policy.h"
+
+#define LOG_MODULE_NAME power
+#define LOG_LEVEL CONFIG_PM_LOG_LEVEL /* From power module Kconfig */
+#include <logging/log.h>
+LOG_MODULE_DECLARE();
+
+#ifdef CONFIG_TICKLESS_KERNEL
+#define SECS_TO_TICKS		1000
+#else
+#define SECS_TO_TICKS		100
+#endif
+
```

Comment:
Isn't the ticks configurable? So, shouldn't you use a macro function to address it here?